source/components/atoms/Input/Input.tsx

```diff
@@ -42,6 +42,7 @@ const StyledErrorText = styled(Text)`
 const Input: React.FC<InputProps> = React.forwardRef(
   ({ onBlur, showErrorMessage, value, error, ...props }, ref) => {
 
+    console.log(value);
```

Comment:
Could we remove the console log?

Reply:
Whoops, of course, that is not intended.

Reply:
Fixed